hnn_core/feed.py

```diff
@@ -179,6 +179,14 @@ def __create_extgauss(self):
     def __create_extinput(self):
         """Creates the ongoing external inputs (rhythmic)."""
         # print("__create_extinput")
+        # Escape if all synaptic weights are 0
+        for key in self.p_ext.keys():
+            if key.startswith('L2Pyr') or \
+                    key.startswith('L5Pyr') or \
+                    key.startswith('L2Bask') or \
+                    key.startswith('L2Bask'):
+                if self.p_ext[key][0] <= 0.0:
+                    return False
```

Comment:
```suggestion
                    return
```

Reply:
Is there a particular reason why it should return nothing? Note that checks for other input types in this function return False.

Reply:
well if the `False` value is not used anywhere, it should return nothing. But agree that it should be consistent